src/chrome/content/rules/AJC.com.xml

```diff
@@ -1,27 +1,17 @@
 <!--
-	^ajc.com: Handshake fails
-	www.ajc.com: 504, Akamai
-
-
-	Problematic hosts in *ajc.com:
-
-		- *.blog *
-
-	* Mismatched
+	Other Atlanta Journal Constitution rulesets:
+		* MyAJC.com.xml
 
+	Numerous unsupported subdomains (>300 found by Sublist3r)
+	Mixed content on some www.ajc.com pages
 -->
-<ruleset name="AJC.com (partial)" default_off="mismatched">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="*.blog.ajc.com" />
-
-		<test url="http://music.blog.ajc.com/" />
-		<test url="http://newstome.blog.ajc.com/" />
-
-
-	<rule from="^http:"
-		to="https:" />
-
+<ruleset name="AJC.com (partial)">
+	<target host="ajc.com" />
+	<target host="www.ajc.com" />
+	<target host="myaccount.ajc.com" />
+	<target host="store.ajc.com" />
+	<target host="subscribe.ajc.com" />
+
+	<rule from="^https?://ajc\.com/" to="https://www.ajc.com/"/>
```

Comment:
Huh. I can find a number of rulesets (78 cases in 69 rulesets) that have rule(s) that touch https. What I was solving was that when you request a page over https on www.ajc.com, it often seems to add https to other ajc.com links on that page regardless of whether the subdomain supports https. This can be a problem because it will create links like https://ajc.com/news/breaking-news/breaking-house-passes-obamacare-repeal-after-months-infighting/5oBa0i6IOViIvx2FPISdnJ/ which when clicked results in a invalid cert error. The rewriting fixes this by adding the www. which does support https. 

So if you don't know of any way to do this in an acceptable way then I guess I'll just close this PR because the other subdomains I added default to https and the real value added here is encrypting the www. requests. And then I guess we need to created an issue that fixes all the existing rulesets that have rules that touch https.